test/Conversion/ExportVerilog/hw-dialect.mlir

```diff
@@ -849,6 +849,45 @@ hw.module @structExtractFromTemporary(%cond: i1, %a: !hw.struct<c: i1>, %b: !hw.
     hw.output %1 : i1
 }
 
+// Rename field names
+// CHECK-LABEL: renameKeyword(
+// CHECK-NEXT:  input  struct packed {logic repeat_0; logic repeat_0_1; } a,
+// CHECK-NEXT:  output struct packed {logic repeat_0; logic repeat_0_1; } r1);
+hw.module @renameKeyword(%a: !hw.struct<repeat: i1, repeat_0: i1>) -> (r1: !hw.struct<repeat: i1, repeat_0: i1>){
+  hw.output %a : !hw.struct<repeat: i1, repeat_0: i1>
+}
+
+// CHECK-LABEL: externalRenameKeyword
+hw.module.extern @externalRenameKeyword(%a: !hw.struct<repeat: i1, repeat_0: i1>) -> (r:i1)
```

Comment:
Any renaming (or non-renaming) around this doesn't seem to be checked in the test?

Reply:
Thanks, it makes sense. The current implementation rewrites external module declarations as well by mistake. I'll change it to an error.